### PR TITLE
fix: REPLAT-11684 handle responsive-graphic specially on ios

### PR DESCRIPTION
**Pull request by jmars**
This is for https://nidigitalsolutions.jira.com/browse/REPLAT-11684 again.
Reloading on errors did not fix the issue and it appears to be something about ios webviews. This adds a small component which does what the interactive does except without using webviews (as recommended by Chris Hutchinson).

### CODE REVIEW

File: packages/interactive-wrapper/src/responsive-image.ios.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  componentDidMount() {
+    const { deckId } = this.props;
+    if (deckId) {
+      this.getDeck(deckId);
+    }
+  }
+
+  async getDeck(id) {
+    // eslint-disable-next-line no-undef
+    const res = await fetch(deckUrl(id), {
```
Comment (ValGeorgiev) [inline packages/interactive-wrapper/src/responsive-image.ios.js]: Do we have fetch on iOS?

File: packages/interactive-wrapper/src/responsive-image.ios.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        "Content-Type": "application/json"
+      }
+    });
+    const {
+      body: { data }
+    } = await res.json();
+    const images = data
+      .filter(dat => dat.type === "image")
+      .map(d => d.data)
+      .map(d => ({
```
Comment (ValGeorgiev) [inline packages/interactive-wrapper/src/responsive-image.ios.js]: we can combine these maps in one

File: packages/interactive-wrapper/src/responsive-image.ios.js (hunk 1 above)
Comment (jmars) [reply packages/interactive-wrapper/src/responsive-image.ios.js]: Yep fetch is available on ios in the version of JSC that RN uses :)

File: packages/interactive-wrapper/src/responsive-image.ios.js (hunk 2 above)
Comment (jmars) [reply packages/interactive-wrapper/src/responsive-image.ios.js]: Good point :)

**Pull request merged**